### ISSUE TITLE
Validation errors raised for valid response

### ISSUE THREAD

**Issue by Landcross** [OpenXbox/xbox-webapi-python#40]
When I search the catalog for product ID `9P29XK3N9SVQ` (_Assassin’s Creed Valhalla + Watch Dogs: Legion Bundle_) the following error is raised:

```
pydantic.error_wrappers.ValidationError: 2 validation errors for CatalogResponse
Products -> 0 -> ValidationData -> ValidationResultUri
  none is not an allowed value (type=type_error.none.not_allowed)
Products -> 0 -> DisplaySkuAvailabilities -> 0 -> Sku -> Properties -> FulfillmentType
  none is not an allowed value (type=type_error.none.not_allowed)
```

However, when inspecting the raw api response, it seems to work fine. So, I think the validations are a little too restricted for the above values.

**Comment by Landcross** [comment]
Another validation issue:
`Products -> 0 -> DisplaySkuAvailabilities -> 0 -> Sku -> Properties -> Packages
  none is not an allowed value (type=type_error.none.not_allowed)`

**Comment by Landcross** [comment]
Sorry for the spam :(
Found two more validation errors:
```
Products -> 1 -> DisplaySkuAvailabilities -> 0 -> Sku -> Properties -> FulfillmentData -> WuBundleId
  none is not an allowed value (type=type_error.none.not_allowed)
Products -> 1 -> DisplaySkuAvailabilities -> 0 -> Sku -> Properties -> BundledSkus
  none is not an allowed value (type=type_error.none.not_allowed)
```

**Comment by Landcross** [comment]
I found two more validation issues when running a search for 'Creed' in region 'Argentina':
```
Results -> 3 -> Products -> 0 -> Height
  field required (type=value_error.missing)
Results -> 3 -> Products -> 0 -> Width
  field required (type=value_error.missing)
```

**Comment by Landcross** [comment]
Another one 😅
```
pydantic.error_wrappers.ValidationError: 1 validation error for CatalogResponse
Products -> 0 -> DisplaySkuAvailabilities -> 0 -> Availabilities -> 1 -> OrderManagementData
  field required (type=value_error.missing)
```

**Comment by Landcross** [comment]
```
pydantic.error_wrappers.ValidationError: 2 validation errors for CatalogResponse
Products -> 0 -> MarketProperties -> 0 -> OriginalReleaseDate
  field required (type=value_error.missing)
Products -> 0 -> DisplaySkuAvailabilities -> 0 -> Sku -> MarketProperties -> 0 -> FirstAvailableDate
  invalid datetime format (type=value_error.datetime)
```

**Comment by tuxuser** [comment]
Could you send a PR? Simply edit the fields to be `Optional[type]`

**Comment by Landcross** [comment]
Sure, it should be available now in #41 . I had modified the models.py file locally already anyway.
I'm not exactly too familiar with pull-requests, so I hope it all went well :)
